webui/react/src/e2e/models/components/NavigationSideBar.ts

```diff
@@ -52,21 +52,27 @@ export class NavigationSideBar extends BaseReactFragment {
     parent: this.#nav,
     selector: `a[aria-label="${'Workspaces'}"]`,
   });
-  readonly createWorkspace = new BaseComponent({
+  readonly createWorkspaceFromHover = new BaseComponent({
     parent: this.#nav,
     selector: 'span[aria-label="Create workspace"]',
   });
+  // consider this to be a rowContainer
+  readonly #pinnedWorkspaces = new BaseComponent({
+    parent: this.#nav,
+    selector: '[class*="NavigationSideBar_pinnedWorkspaces"]',
+  });
   /**
    * Returns a representation of a sidebar NavigationItem with the specified label.
    * For example, a workspace pinned to the sidebar is accessible through it's label here.
    * @param {string} label - the label of the tab, generally the name
    */
   public sidebarWorkspaceItem(label: string): SidebarWorkspaceItem {
     return new SidebarWorkspaceItem({
-      parent: this.#nav,
+      parent: this.#pinnedWorkspaces,
       selector: `a[aria-label="${label}"]`,
     });
   }
+  // consider the other add workspace button
```

Comment:
i was too lazy too add the other workspace button, but it'll also have a parent of `this.#pinnedWorkspaces`